config/openhpc.yml

```diff
@@ -120,7 +120,8 @@ beegfs_oss:
 - dev: /dev/md0
   port: 8003
 beegfs_client:
-- path: "/mnt/{{ cluster_name }}"
+#- path: "/mnt/{{ cluster_name }}"
+- path: "{{ alaska_homedir }}"
```

Comment:
What's this?

Reply:
Its the current state of the cluster where I've disabled ceph mount in favour of beegfs storage to be used as home directory

Reply:
Ok. Should really have been in a different commit with reasoning.